releases/release-1.14/README.md

```diff
@@ -78,61 +83,115 @@ The 1.14 release cycle is proposed as follows:
+Starting at [Enhancements Freeze] the release team will solicit and rule on 
+[exception requests] for enhancements and test work that is unlikely to be done
+by Code Freeze. The exception approval is the responsibility of the SIG or SIGs
+labeled in the pull request. The release team may intervene or deny the request 
+only if it poses a risk to release quality, or could negatively impact the overall
+timeline. Changes introduced at this point should be well-tested,
+well-understood, limited in architectural scope, and low risk.  All of those
+factors should be considered in the approval process.  Enhancements on an
+feature branch with documentation, test cases, and passing CI are more likely to
+be accepted.
 
 ### Pruning
 
-Enhancements that are partially implemented and/or lack sufficient tests may be considered for pruning beginning after [Code Freeze], unless they've been granted [Exceptions]
+Enhancements that are partially implemented and/or lack sufficient tests may be
+considered for pruning beginning after [Code Freeze], unless they've been
+granted [Exceptions]
 
-The release team will work with SIGs and enhancements owners to evaluate each case, but for example, pruning could include actions such as:
+The release team will work with SIGs and enhancements owners to evaluate each
+case, but for example, pruning could include actions such as:
 
 * Disabling the use of a new API or field
 * Switching the default value of a flag or field
 * Moving a new API or field behind an Alpha Enhancements gate
 * Reverting commits or deleting code
 
-This should occur before 1.14.0-beta.1 is cut so we have time to gather signal on whether the system is stable in this state. These are considered drastic measures, so the release team will strive to coordinate at-risk work with SIGs before this time. The goal is to make code freeze, and overall project transparency, enforceable despite the lack of a consistently used feature branch process.
+This should occur before 1.14.0-beta.1 is cut so we have time to gather signal
+on whether the system is stable in this state. These are considered drastic
+measures, so the release team will strive to coordinate at-risk work with SIGs
+before this time. The goal is to make code freeze, and overall project
+transparency, enforceable despite the lack of a consistently used feature branch
+process.
```

Comment:
would it be worth mentioning the in-tree vs out-of-tree? out-of-tree being more lenient since it's not touching anything in k/k?